ui/app/components/oidc/assignment-form.js

```diff
@@ -69,6 +71,12 @@ export default class OidcAssignmentFormComponent extends Component {
     }
   }
 
+  async fetchEntitiesAndGroups() {
```

Comment:
I have a `fetchAssignments` method because I need it to determine what radio select button should be selected - I don't think you need to fetch these as well? Or maybe I'm missing something